bftengine/src/preprocessor/PreProcessor.cpp

```diff
@@ -1120,7 +1121,7 @@ uint32_t PreProcessor::launchReqPreProcessing(uint16_t clientId,
       std::move(signature),
       maxPreExecResultSize_,
       (char *)getPreProcessResultBuffer(clientId, reqSeqNum, reqOffsetInBatch)});
-  requestsHandler_.execute(accumulatedRequests, cid, span);
+  requestsHandler_.execute(accumulatedRequests, std::nullopt, cid, span);
```

Comment:
Just curious - why the timestamp is not set in preexecution?

Reply:
The reason is because in pre-execution we cannot guarantee full consensus on the timestamp.

Reply:
Makes sense. Thanks!